.github/dependabot.yaml

```diff
@@ -34,8 +25,81 @@ updates:
   - package-ecosystem: "npm"
     directory: "/docs"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
     groups:
       npm-deps:
         patterns:
           - "*"
+  # Dart Dependencies
+  - package-ecosystem: pub
+    directory: /packages/dart_frog
+    schedule:
+      interval: monthly
+    groups:
+      packages-dart-frog-pub:
```

Comment:
Can we still group them under on dart/pub group to keep the number of dependabot PRs down?